### ISSUE TITLE
easier trading system

### ISSUE THREAD

**Issue by Generalyossef** [naev/naev#2044]
im confused from the trading system (it looks very fine)
but i just like the 
(very low)
(low)
(mid)
(high)
(very high)
next to the price... will make it much easier for braindead traders like me :D

**Comment by not-pyroman** [comment]
The prices currently follow a sinusoidal pattern, and the high low system probably wouldn't work super well, as it could still be in the range of "low" but one spob is asking a higher price than the other

**Comment by bobbens** [comment]
It shouldn't be too hard to calculate the mean of known prices and show deviation from that as either high or low, and that could be quite useful instead of having to memorize numbers.

**Comment by PhoenixRiver** [comment]
I'm confused, doesn't it already show a galactic average price for a commodity below the price at this station? (Though I'm not using the nightlies..., I'm on Steam 0.9.2...)

**Comment by bobbens** [comment]
Yes, it does already show averages and lots more details. Adding something like "low", "medium", or "high" might be helpful for players used to escape velocity that only had 3 hardcoded categories iirc.

**Comment by PhoenixRiver** [comment]
I remember thinking it was simplistic with EV too but I was used to Elite on the BBC ;)

**Comment by agb32** [comment]
It would be straight forward to calculate the mean price seen across the galaxy, and the standard deviation of this, and then represent the current cost at the current location as a very low, low etc, depending on how many standard deviations from the mean the current cost is.
